pkg/model/dev.go

```diff
@@ -128,7 +128,7 @@ type Dev struct {
 	Environment          []EnvVar              `json:"environment,omitempty" yaml:"environment,omitempty"`
 	Secrets              []Secret              `json:"secrets,omitempty" yaml:"secrets,omitempty"`
 	Command              Command               `json:"command,omitempty" yaml:"command,omitempty"`
-	Healthchecks         bool                  `json:"healthchecks,omitempty" yaml:"healthchecks,omitempty"`
```

Comment:
Keep `Healthchecks` for backward compatibility, and define a new field `Probes`

Reply:
If `healthchecks` is true, then set all probes to true.
Also, if all probes are `true`, set `healthchecks` to true.

Reply:
there is a healthcheck field that if you can set a boolean value that sets all the probes to true (backwards compatibility), you can also specify in that healthchecks field the probes that want to be enabled by specifying which one should be enabled.